### PR TITLE
Add css comb

### PR DESCRIPTION
**Pull request by wfortin**
CSS Comb settings following codeguide.co rules.
Changes include settings for CSSComb, a gulp task to run comb, and modified files with new code style.

I'll add a watcher to run this on save.

I suggest checking the review with "split" view.

### CODE REVIEW

**Comment by JPLachance** [comment]
Ship it!

File: less/controls/checkboxes.less [hunk 1]
```diff
@@ -40,107 +40,111 @@
...
+
+        box-sizing: content-box;
+        display: block;
+        // Make it a small rectangle so the border will create an L-shape
+        width: 4px;
+        height: 8px;
+
+        content: '';
+        // Add a white border on the bottom and left, creating that 'L'
+
```
Comment (Unetitebiere) [inline less/controls/checkboxes.less]: Enlever l'espace

Comment (wfortin) [inline less/controls/checkboxes.less]: Not sure i understand...

File: package.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
     "gulp-autoprefixer": "3.0.2",
     "gulp-cheerio": "0.6.2",
     "gulp-concat": "2.6.0",
+    "gulp-csscomb": "^3.0.6",
```
Comment (marilynechan) [inline package.json]: On veut vraiment ^ ?

File: less/controls/checkboxes.less (hunk 1 above)
Comment (marilynechan) [inline less/controls/checkboxes.less]: Comments look a bit mixed up because of the properties reordering.

Comment (wfortin) [inline less/controls/checkboxes.less]: Oh ok i get it, it seems that CSSComb does not keep the comments order, making it unrelated to the property. I'll fix that.

**Comment by pastjean** [comment]
Would be nice if the build failed when the files checked-in aren't combed

1. Add a task testComb to gulp
2. Compare files
3. fail if they are different

**Comment by marilynechan** [comment]
We already talked about it, but it would also be nice if the css was *combed* on save.
Otherwise, Ship It!

**Comment by Unetitebiere** [comment]
LGTM! Ship it

**Comment by pastjean** [comment]
+1 @marilynechan watch comb task + build fail task if someone forgets the watch save , otherwise :shipit:

**Pull request merged**